<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSatuan</class>
 <widget class="QWidget" name="FormSatuan">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>500</width> <height>400</height> </rect> </property>
  <property name="windowTitle"> <string>Data Satuan</string> </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry"> <rect> <x>50</x> <y>20</y> <width>400</width> <height>100</height> </rect> </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0"><widget class="QLabel" name="l1"><property name="text"><string>ID Satuan</string></property></widget></item>
    <item row="0" column="1"><widget class="QLineEdit" name="editID"/></item>
    <item row="1" column="0"><widget class="QLabel" name="l2"><property name="text"><string>Kode</string></property></widget></item>
    <item row="1" column="1"><widget class="QLineEdit" name="editKode"/></item>
    <item row="2" column="0"><widget class="QLabel" name="l3"><property name="text"><string>Nama Satuan</string></property></widget></item>
    <item row="2" column="1"><widget class="QLineEdit" name="editNama"/></item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan"> <property name="geometry"> <rect> <x>60</x> <y>140</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Simpan</string> </property> </widget>
  <widget class="QPushButton" name="btnHapus"> <property name="geometry"> <rect> <x>170</x> <y>140</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Hapus</string> </property> </widget>
  <widget class="QPushButton" name="btnUbah"> <property name="geometry"> <rect> <x>280</x> <y>140</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Ubah</string> </property> </widget>
  <widget class="QPushButton" name="btnBersih"> <property name="geometry"> <rect> <x>390</x> <y>140</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Bersih</string> </property> </widget>
  <widget class="QTableWidget" name="tabelSatuan">
   <property name="geometry"> <rect> <x>50</x> <y>190</y> <width>430</width> <height>200</height> </rect> </property>
   <column><property name="text"><string>ID</string></property></column>
   <column><property name="text"><string>Kode</string></property></column>
   <column><property name="text"><string>Nama</string></property></column>
  </widget>
 </widget>
 <resources/> <connections/>
</ui>
